node/docs/as-json.md

```diff
@@ -111,13 +60,8 @@ send call requests and register call request handlers
 
 It can be passed options.
 
- - `opts.logger` is a logger passed in. `TChannelJSON` writes
-    a series of log warnings and errors under various JSON
-    serialization failures.
- - `opts.strictMode` strictMode defaults to true. When strictMode
-    is set to true we enforce that a not ok response from a handler
-    must have the body set to an instance of `TypedError` rather
-    then arbitary response.
+ - `opts.bossMode` bossMode defaults to false. When bossMode
```

Comment:
Not sure if we really need this bossMode. If it is critical we check it; otherwise, we warn it. From what I see in the code, warning might be the best choice.